### PR TITLE
CSP-846 CalendarEventDetails page now has google maps link

### PR DESCRIPTION
**Pull request by MarkFCain**

### CODE REVIEW

File: src/SFA.DAS.ApprenticeAan.Web/Views/NetworkEventDetails/_InPersonEventPartial.cshtml [hunk 1]
```diff
@@ -4,13 +4,9 @@
...
 @model NetworkEventDetailsViewModel
 
-<div class="govuk-grid-column-one-half">
     <h2 class="govuk-heading-m">When and where</h2>
-    <p>@Model.StartDate<span class="das-!-display--block">@Model.StartTime to @Model.EndTime</span> </p>
+    <p><b>Date and Time:</b> @Model.StartDate @Model.StartTime to @Model.EndTime</p>
     <p>
-        In person <br />@Model.LocationDetails?.Location, <span class="das-no-wrap">@Model.LocationDetails?.Postcode</span>
+        <b>In person:</b> @Model.LocationDetails?.Location, <span class="das-no-wrap">@Model.LocationDetails?.Postcode</span>
+        <br /><a href="https://www.google.com/maps/dir//@Model.LocationDetails?.Location+@Model.LocationDetails?.Postcode" rel="noopener" target="_blank" class="govuk-link">Find the location on google maps (opens in a new window)</a>
```
Comment (DevenKShah) [inline src/SFA.DAS.ApprenticeAan.Web/Views/NetworkEventDetails/_InPersonEventPartial.cshtml:11]: Can the concatenation be moved to the model? also there is no delimiter between location and postcode

File: src/SFA.DAS.ApprenticeAan.Web/Views/NetworkEventDetails/_HybridEventPartial.cshtml [hunk 2]
```diff
@@ -4,14 +4,10 @@
...
 
-<div class="govuk-grid-column-one-half">
     <h2 class="govuk-heading-m">When and where</h2>
-    <p>@Model.StartDate<span class="das-!-display--block">@Model.StartTime to @Model.EndTime</span> </p>
+    <p><b>Date and Time:</b> @Model.StartDate @Model.StartTime to @Model.EndTime</p>
     <partial name="_OnlineJoiningDetails" />
     <p>
-        In person <br />@Model.LocationDetails?.Location, <span class="das-no-wrap">@Model.LocationDetails?.Postcode</span>
+        <b>In person:</b> @Model.LocationDetails?.Location, <span class="das-no-wrap">@Model.LocationDetails?.Postcode</span>
+    <br /><a href="https://www.google.com/maps/dir//@Model.LocationDetails?.Location+@Model.LocationDetails?.Postcode" rel="noopener" target="_blank" class="govuk-link">Find the location on google maps (opens in a new window)</a>
```
Comment (DevenKShah) [inline src/SFA.DAS.ApprenticeAan.Web/Views/NetworkEventDetails/_HybridEventPartial.cshtml:12]: If we can calculate the whole URL in the model, then we can test it and the logic of calculating this url remains in one place. Currently I can see this is in two different places.

**Review by DevenKShah — APPROVED** [review]
Approving, however just one suggestion to move the url calculation into the model to avoid duplication and also easy binding in the view.

File: src/SFA.DAS.ApprenticeAan.Web/Views/NetworkEventDetails/_InPersonEventPartial.cshtml (hunk 1 above)
Comment (MarkFCain) [reply src/SFA.DAS.ApprenticeAan.Web/Views/NetworkEventDetails/_InPersonEventPartial.cshtml:11]: sure.  The + acts as the delimiter

**Review by DevenKShah — APPROVED** [review]

**Pull request merged**